File: plugins/particle-auth/android/src/main/kotlin/network/particle/flutter/bridge/model/FlutterErrorMessage.java
package network.particle.flutter.bridge.model;

import com.google.gson.annotations.SerializedName;

public class FlutterErrorMessage {
    public static final int ExceptionCode = 10000;
    @SerializedName("code")
    public int code;

    @SerializedName("message")
    public String message;


    public FlutterErrorMessage(int code, String message) {
        this.code = code;
        this.message = message;
    }

    public FlutterErrorMessage() {
    }

    public static FlutterErrorMessage exceptionMsg(String message) {
        return new FlutterErrorMessage(ExceptionCode, message);
    }


}

File: particle-auth-core/android/src/main/kotlin/network/particle/auth_core_flutter/bridge/model/FlutterErrorMessage.kt
package network.particle.auth_core_flutter.bridge.model

import com.google.gson.annotations.SerializedName

class FlutterErrorMessage {
    @SerializedName("code")
    var code = 0

    @SerializedName("message")
    var message: String? = null

    constructor(code: Int, message: String?) {
        this.code = code
        this.message = message
    }

    constructor() {}

    companion object {
        const val ExceptionCode = 10000
        fun exceptionMsg(message: String?): FlutterErrorMessage {
            return FlutterErrorMessage(ExceptionCode, message)
        }
    }
}
